presto-cassandra/src/test/resources/cu-cassandra.yaml

```diff
@@ -94,10 +94,10 @@ partitioner: org.apache.cassandra.dht.Murmur3Partitioner
 
 # directories where Cassandra should store data on disk.
 data_file_directories:
-    - target/embeddedCassandra/data
+    - ${data_directory}/data
```

Comment:
`cassandra-unit` was doing all the `target/embeddedCassandra` magic. I'm not so smart, so i decided to go with more obvious way, and create temprorary directory manually. If it is created manually - it must be specified manually.